examples/advanced/tidb-cluster.yaml

```diff
@@ -1,295 +1,872 @@
+    ## The desired replicas
     replicas: 3
 
-    # describes the compute resource requirements.
-    # Ref: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
-    requests:
-      cpu: "100m"
-      storage: 1Gi
-
-    # TiDB Operator will automatically mount the certificates
-    # Defaults to "false"
-    # Ref: https://docs.pingcap.com/tidb-in-kubernetes/stable/configure-a-tidb-cluster#mountclusterclientsecret
-    mountClusterClientSecret: false
-    
+    ## max inprogress failover PD pod counts
+    # maxFailoverCount: 3
+
+    ## describes the compute resource requirements and limits.
+    ## Ref: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
+    # requests:
+    #   cpu: 1000m
+    #   memory: 1Gi
+    #   storage: 1Gi
+    # limits:
+    #   cpu: 2000m
+    #   memory: 2Gi
+
+    ## defines Kubernetes service for pd-server
+    ## Ref: https://kubernetes.io/docs/concepts/services-networking/service/
+    # service:
+    #   type: ClusterIP
+    #   annotations:
+    #     foo: bar
+    #   portName: client
+
+    #############################
+    # Advanced PD Configuration #
+    #############################
+
+    ## The following block overwrites TiDB cluster-level configurations in `spec`
+    # version: "v4.0.8"
+    # imagePullPolicy: IfNotPresent
+    # imagePullSecrets: secretName
+    # hostNetwork: false
+    # serviceAccount: advanced-tidb-pd
+    # priorityClassName: system-cluster-critical
+    # schedulerName: tidb-scheduler
+    # nodeSelector:
+    #   app.kubernetes.io/component: pd
+    # annotations:
+    #   node.kubernetes.io/instance-type: some-vm-type
+    # tolerations:
+    #   - effect: NoSchedule
+    #     key: dedicated
+    #     operator: Equal
+    #     value: pd
+    # configUpdateStrategy: RollingUpdate
+    # statefulSetUpdateStrategy: RollingUpdate
+
+    ## List of environment variables to set in the container
+    ## Note that the following env names cannot be used and will be overwritten by TiDB Operator builtin envs
+    ##   - NAMESPACE
+    ##   - TZ
+    ##   - SERVICE_NAME
+    ##   - PEER_SERVICE_NAME
+    ##   - HEADLESS_SERVICE_NAME
+    ##   - SET_NAME
+    ##   - HOSTNAME
+    ##   - CLUSTER_NAME
+    ##   - POD_NAME
+    ##   - BINLOG_ENABLED
+    ##   - SLOW_LOG_FILE
+    ## Ref: https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/
+    # env:
+    # - name: MY_ENV_1
+    #   value: value1
+    # - name: MY_ENV_2
+    #   valueFrom:
+    #     fieldRef:
+    #       fieldPath: status.myEnv2
+
+    ## Custom sidecar containers can be injected into the PD pods,
+    ## which can act as a logging/tracing agent or for any other use case
+    # additionalContainers:
+    # - name: myCustomContainer
+    #   image: ubuntu
+
+    ## custom additional volumes in PD pods
+    # additionalVolumes:
+    # # this must match `name` in `additionalVolumeMounts`
```

Comment:
This is not the case, maybe only part of the volumes will be mounted to the PD pods.
Similar to tidb and tikv.

Reply:
I see, volumes can be specified but not mounted.